src/Extension/Context.ts

```diff
@@ -19,27 +20,125 @@ export function GetContext(): Contexts {
         if (scheme || location.hostname === 'localhost') {
```

Comment:
```suggestion
        if (scheme || location.hostname === 'localhost'|| location.hostname === '127.0.0.1') {
```

Reply:
`GetContext()` is deprecated and not introducing new changes.